packages/fxa-content-server/app/scripts/templates/pair/failure.mustache

```diff
@@ -1,18 +1,18 @@
-<div class="card">
-  <header>
-    <h1 id="fxa-pair-failure-header" class="card-header">
-      {{#t}}Pairing not successful{{/t}}
-    </h1>
-  </header>
-
+<div class="card-base">
   <section>
     <div class="error"></div>
     <div class="success"></div>
 
     <form novalidate>
       <div class="bg-image-pair-fail"></div>
 
-      <p>{{#t}}The setup process was terminated. Please sign in with your email.{{/t}}</p>
+      <header>
+        <h1 id="fxa-pair-failure-header" class="card-header">
+          {{#t}}Device pairing failed{{/t}}
+        </h1>
+      </header>
+
+      <p class="mt-3">{{#t}}The setup couldn’t be completed.{{/t}}{{#showSigninLink}} {{#t}}Please sign in with your email.{{/t}}{{/showSigninLink}}</p>
```

Comment:
QA recently reopened https://mozilla-hub.atlassian.net/browse/FXA-10258 and I have been stratching my head on this. I think the best thing to do is to remove the button completely and tell the user to login with their email.

Reply:
We should also be able to close the above ticket as well.

Reply:
Hmm was that a requested feature? If we shouldn't spend more time trying to fix it (I'm also not sure what's going on without investigating, I see the params are passed), I'm OK with removing it and revisiting when we do React work on that page.

Reply:
Chatted in Slack and Vijay will remove this with the linked ticket in a quick follow up after this lands.